reference/spender.yaml

```diff
@@ -2450,6 +2450,54 @@ components:
             - ROUND_TRIP
             - NO_DEDUCTION
             - null
+    spender_expenses_in:
+      type: array
+      additionalProperties: false
+      items:
+        type: object
+        required:
+          - id
+        properties:
+          id:
+            allOf:
+              - $ref: '#/components/schemas/id_string'
+            description: |
+              This id is provided by Fyle to identify an object
+          merchant:
+            $ref: '#/components/schemas/merchant'
+          purpose:
+            type: string
+            example: Team lunch
+          cost_center_id:
+            allOf:
+              - $ref: '#/components/schemas/fk_integer'
+            nullable: true
+            example: 1049
+          category_id:
+            allOf:
+              - $ref: '#/components/schemas/fk_integer'
+            nullable: true
+            example: 49058
+          project_id:
+            allOf:
+              - $ref: '#/components/schemas/fk_integer'
+            nullable: true
+            example: 92895
+          source_account_id:
+            allOf:
+              - $ref: '#/components/schemas/fk_string'
+            example: acwbl222wlg
+          advance_wallet_id:
+            allOf:
+              - $ref: '#/components/schemas/fk_string'
+            nullable: true
+            example: advwrlAGwjr2
+          is_billable:
+            $ref: '#/components/schemas/is_billable'
+          is_reimbursable:
+            $ref: '#/components/schemas/is_reimbursable'
+          custom_fields:
+            $ref: '#/components/schemas/custom_fields'  
```

Comment:
I have added source_account_id,  advance_wallet_id, and is_reimbursable: because we want to support the payment mode to be updated in bulk(that would require these keys).